app/_includes/md/enterprise/deploy-license.md

```diff
@@ -6,8 +6,11 @@ It has one parameter include.heading which must be included when used.  -->
 
 You can deploy a license file in one of the following ways:
 
-* Through the `/licenses` Admin API endpoint.
-* As a file on the node filesystem.
+Method | Supported deployment types
+-------|---
+ `/licenses` Admin API endpoint | &#8226; Traditional database-backed deployment <br> &#8226; Hybrid mode deployment
+File on the node filesystem <br>(`license.json`) | &#8226; Traditional database-backed deployment <br> &#8226; DB-less mode
+Environmental variable<br>(`KONG_LICENSE_DATA`) | &#8226; Traditional database-backed deployment <br> &#8226; DB-less mode
```

Comment:
`KONG_LICENSE_PATH` is also method for deploying the license locally to either a CP or DP

```
Environmental variable<br>(`KONG_LICENSE_PATH`) | &#8226; Traditional database-backed deployment <br> &#8226; DB-less mode
```

This environment variable is nothing more than another way to point to a local file that contains the license is JSON format. It can be named anything, it can be located anywhere (of course filesystem permissions apply here), and its contents are exactly the same as `/etc/kong/license.json`.